src/odemis/driver/actuator.py

```diff
@@ -2405,27 +2401,18 @@ def _checkRxMovement(self, pos, rel=False):
         - Otherwise (ie, moving Rx with focus active), raise a ValueError
         :param pos: The target position
         :param rel: whether it's a relative movement or absolute
-        :return: (bool) whether there is a movement in Rx axis or not
         :raises: (ValueError) when moving Rx with focus active
         """
+        if "rx" not in pos:
+            return
         current_rx = self.position.value['rx']
-        if not util.almost_equal(current_rx, 0, atol=1e-3):
-            # Stage Rx is already not 0 => Do not move focus
-            return False
-        target_rx = pos['rx'] if 'rx' in pos else None
-        if target_rx is None:
-            return True  # No movement found in Rx => Save to move
+        target_rx = pos['rx']
         if rel:
             target_rx += current_rx  # To compare relative value
-
-        if util.almost_equal(current_rx, target_rx, atol=1e-3):
-            return True  # Both current and target is 0 => Save to move
-        else:
-            # Only allow a movement in Rx when focus is not in active range
-            if self._focus._isInRange():
-                raise ValueError("Movement in Rx is not allowed while focus is in active position.")
-            else:
-                return False  # Target Rx is not 0 => Do not move focus
+        if util.almost_equal(current_rx, target_rx, atol=ATOL_LENS_POS):
+            return  # Not moving
+        if self._focus._isInRange():
```

Comment:
I think it'd be a tiny be safer to use `_isParked()`, and raise a ValueError() whenever it's not officially at the parked position.